install/operator/pkg/syndesis/action/checkupdates.go

```diff
@@ -40,15 +41,11 @@ func (a checkUpdatesAction) Execute(ctx context.Context, syndesis *v1alpha1.Synd
 	} else {
 		// Let's start the upgrade process
 		target := syndesis.DeepCopy()
-		target.Status.Phase = v1alpha1.SyndesisPhaseUpgrading
-		target.Status.TargetVersion = a.operatorVersion
-		target.Status.Reason = v1alpha1.SyndesisStatusReasonMissing
-		target.Status.Description = "Upgrading from " + syndesis.Status.Version + " to " + a.operatorVersion
-		target.Status.LastUpgradeFailure = nil
-		target.Status.UpgradeAttempts = 0
-		target.Status.ForceUpgrade = false
-
-		a.log.Info("Starting upgrade of Syndesis resource", "name", syndesis.Name, "currentVersion", syndesis.Status.Version, "targetVersion", a.operatorVersion, "type", "checkUpdate")
-		return a.client.Update(ctx, target)
+		syndesis.Status.Phase = v1alpha1.SyndesisPhaseAttachingVolume
+		syndesis.Status.PreviousPhase = target.Status.Phase
+		syndesis.Status.Reason = v1alpha1.SyndesisStatusReasonMissing
+		syndesis.Status.Description = ""
```

Comment:
Perhaps it would be good for troubleshooting to have a description here.

Reply:
included in https://github.com/syndesisio/syndesis/pull/7358/commits/1b84f97655e2d61cf1e662050b9deb4af7db6544